README.rst

```diff
@@ -8,6 +8,14 @@ Svglib
 A pure-Python library for reading and converting SVG
 ---------------------------------------------------------------------------
 
+|ci| |appveyor|
+
+.. |ci| image:: http://img.shields.io/travis/deeplook/svglib.svg
+  :target: https://travis-ci.org/deeplook/svglib
+
+.. |appveyor| image:: https://ci.appveyor.com/api/projects/status/pid1t7iuwhkm9eh6/branch/master?svg=true
```

Comment:
This URL is not correct yet, it should be updated once AppVeyor is enabled for `deeplook/svglib`